contracts/container/contract.go

```diff
@@ -444,7 +444,9 @@ func PutNamed(container []byte, signature interop.Signature,
 	}
 
 	runtime.Log("added new container")
-	runtime.Notify("PutSuccess", containerID, publicKey)
+	h := crypto.Sha256(publicKey)
+	acc := crypto.Ripemd160(h[:])
```

Comment:
Better just emit its owner. `publicKey` can be null eventually, I think we can avoid using it.

Reply:
owner diffs from signer when session token is provided. We wanna throw owner in this case too?

Reply:
done with container owner